src/components/tutorialEditor/index.tsx

```diff
@@ -135,12 +307,148 @@ const TutEditor: React.FC = (): JSX.Element => {
             <StyledTitle>
               <h3>Step Content</h3>
             </StyledTitle>
+            <StepContentBody>
+              <StepContentList>
```

Comment:
For the sake of trimming down the amount of lines, you could move `StepContent` into it's own component.